src/osm_poi_categories.json

```diff
@@ -0,0 +1,12 @@
+{
+    "health": ["clinic", "dentist", "doctors", "hospital","optician", "pharmacy"],
+    "crit_lifelines": ["atm","bank", "college", "community_centre", "comms_tower", "convenience",
```

Comment:
seems like 'crit_lifelines' is not the optimal label for this subset of poi categories. 

- move 'comms_tower', 'telephone' to `telco`  label
- remove tower, already in `power` category
- add new label `commercial`: ['atm', 'bank', 'convenience ;store', 'market_place', 'supermarket'] 
- add new label `public:` [remaining categories in crit_lifelines]